migrations/versions/201608011329_4bf6f41baafa_add_notification_templates_models.py

```diff
@@ -0,0 +1,43 @@
+"""Add Abstracts Notification Templates models
+
+Revision ID: 4bf6f41baafa
+Revises: 23ef6a49ae28
+Create Date: 2016-08-01 13:29:21.978217
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+
+# revision identifiers, used by Alembic.
+revision = '4bf6f41baafa'
+down_revision = '23ef6a49ae28'
+
+
+def upgrade():
+    op.create_table(
+        'notification_templates',
+        sa.Column('include_authors', sa.Boolean(), nullable=False),
+        sa.Column('include_submitters', sa.Boolean(), nullable=False),
+        sa.Column('include_coauthors', sa.Boolean(), nullable=False),
+        sa.Column('position', sa.Integer(), nullable=False),
+        sa.Column('body', sa.Text(), nullable=False),
+        sa.Column('title', sa.String(), nullable=False),
+        sa.Column('event_id', sa.Integer(), nullable=False),
+        sa.Column('reply_to_address', sa.String(), nullable=False),
+        sa.Column('extra_cc_emails', postgresql.ARRAY(sa.String()), nullable=True),
+        sa.Column('stop_on_match', sa.Boolean(), nullable=False),
+        sa.Column('conditions', postgresql.JSON(), nullable=False),
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('subject', sa.String(), nullable=False),
+        sa.ForeignKeyConstraint(['event_id'], [u'events.events.id'],
+                                name=op.f('fk_notification_templates_event_id_events')),
+        sa.PrimaryKeyConstraint('id', name=op.f('pk_notification_templates'))
+    )
+    op.create_index(op.f('ix_notification_templates_event_id'), 'notification_templates', ['event_id'], unique=False)
+
+
+def downgrade():
+    op.drop_index(op.f('ix_notification_templates_event_id'), table_name='notification_templates')
+    op.drop_table('notification_templates')
```

Comment:
`schema='event_abstracts'` missing here and during event creation (it'd have been there had the tableargs been set correctly ;))